taichi/backends/vulkan/vulkan_device_creator.cpp

```diff
@@ -156,16 +156,32 @@ VulkanQueueFamilyIndices find_queue_families(VkPhysicalDevice device,
   return indices;
 }
 
-bool is_device_suitable(VkPhysicalDevice device, VkSurfaceKHR surface) {
+size_t is_device_suitable(VkPhysicalDevice device, VkSurfaceKHR surface) {
```

Comment:
can we rename this to a better name, e.g. `get_device_score`, it feels a bit strange to return a size_t for `is_device_suitable` :P